.github/workflows/e2e_openssl.yml

```
@@ -60,4 +61,11 @@ jobs:
           just coordinator initializer openssl port-forwarder node-installer
       - name: E2E Test
         run: |
-          nix shell .#contrast.e2e --command openssl.test -test.v workspace/just.containerlookup
+          nix shell .#contrast.e2e --command openssl.test -test.v \
```

```suggestion
          nix shell .#contrast.e2e --command openssl.test -test.v -args \
```

that should fix the errors

---

The `-args` is only a flag when using `go test`, I think. The problem is that the input boolean gets parsed as an empty string and not as `false`.

---

Oh, I see.  How about `--skip-undeploy="${{ inputs.skip-undeploy && "true" || "false" }}"`?